README.md

```diff
@@ -1 +1,20 @@
-# semaphore-stellar
\ No newline at end of file
+# semaphore-stellar
+This project aims to integrate Semaphore's privacy-focused protocol with the Stellar network, enabling private and anonymous voting or signaling in Stellar-based applications. Since Stellar is not EVM-compatible, this tool bridges the gap to make Semaphore functionalities accessible in Stellar's ecosystem.
+
+## Project Structure
+- Documentation: Key technical documents and guides are located in the root directory and the [`research`](https://github.com/ZencypherSolutions/semaphore-stellar-docs/tree/main/research) folder.
+- Contracts: Smart contracts designed for Semaphore-Stellar interaction are within the [contracts](./semaphore-contracts/contracts) folder.
+
+## About Semaphore
+Semaphore is a zero-knowledge protocol designed to provide privacy for applications. It allows users to prove membership in a group without revealing their identity, making it ideal for anonymous voting or signaling.
+- Official Semaphore Project: [Semaphore Protocol](https://semaphore.pse.dev/)
+- Source Code: [Semaphore GitHub Repository](https://github.com/semaphore-protocol/semaphore)
+
+## Community Support
+For questions, you may join these groups:
+- Semaphore-Stellar Maintainers: [Join the Telegram Group](https://t.me/+-9623JNgGjEyNzI5)
+- Semaphore Community (core developers): Search @semaphore_community on Telegram.
```

Comment:
Could you please add that we're not linked to Semaphore Core developers but that the group can be used for direct support.

Reply:
I chose to delete that line.